ncm-network/src/main/perl/nmstate.pm

```diff
@@ -296,6 +296,26 @@ sub generate_vip_config {
     return $iface_cfg;
 }
 
+# find vlan id from either the name or device.
+# i.e eth0.123 will return 123. by checking iface name and device
+# returns vlan id.
+sub get_vlanid(){
```

Comment:
I assume this works but replacing the () with a space would be consistent with line 278?